src/Faker/Name.cs

```diff
@@ -46,6 +46,15 @@ public static class Name
 		/// <returns>The randomly created first name.</returns>
 		public static string First()
 		{
+			return Resources.Name.First.Split(Config.SEPARATOR).Random().Trim();
+		}
+
+		/// <summary>
+		///   Creates a random first name, using a more aggressive caching strategy.
+		/// </summary>
+		/// <returns>The randomly created first name.</returns>
+		public static string FirstCached()
```

Comment:
yeah, you are entirely correct.
Scratch what I said earlier, let's just go with the single method